core/modules/qana/ScanTablePlugin.cc

```diff
@@ -184,7 +184,7 @@ ScanTablePlugin::_findScanTables(query::SelectStmt& stmt,
     }
 
     auto czar = czar::Czar::getCzar();
-    bool queryDistributionTestVer = czar->getQueryDistributionTestVer();
+    bool queryDistributionTestVer = (czar == nullptr) ? 0 : czar->getQueryDistributionTestVer();
```

Comment:
Is this meant to address issues with unit testing?

Reply:
Yes